en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md

```diff
@@ -0,0 +1,57 @@
+# Connecting to Azure Service Bus
+
+This section describes how to configure WSO2 Micro Integrator to connect with Azure Service Bus.
+
+The integration between this message broker and WSO2 Micro Integrator is straightforward. The guaranteed delivery integration pattern must be used to implement the mediation flow.
+
+## Setting up the Micro Integrator with Azure Service Bus
+
+Follow the instructions below to set up and configure.
+
+1.  Download [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/).
+2.  Download and install WSO2 Micro Integrator.
+
+### Setting up Azure Service Bus
```

Comment:
Azure Service Bus configuration must go here.

Reply:
Added new content here.

Reply:
We have to mentioned how to configure a queue in Azure Service Bus to work with the following synapse config. Can you add a small paragraph and give this link? https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal

Reply:
Fixed.